build/bin/build-server-rpm

```diff
@@ -206,8 +206,8 @@ d }\" \
                 spectool -C SOURCES -g SOURCES/${spec_name}.spec
 
                 sudo go version
-                sudo wget https://dl.google.com/go/go1.11.5.linux-amd64.tar.gz
-                sudo tar -C /usr/local -xzf go1.11.5.linux-amd64.tar.gz
+                sudo wget https://dl.google.com/go/go1.12.1.linux-amd64.tar.gz
```

Comment:
Could you leave it as 1.11.5 ?
The latest version isn't working smoothly with grafana.

Reply:
this question is not to me :) It was requested to get 1.12.1

Reply:
@rnovikovP could you please check and decide

Reply:
In general I think we should check latest grafana and fix qan2

Reply:
The latest Grafana build has no fix for the issue.